doc/src/manual/integers-and-floating-point-numbers.md

```diff
@@ -612,6 +612,12 @@ julia> 2(x-1)^2 - 3(x-1) + 1
 3
 ```
 
+!!! warning
+    Notice that, in this case, the precedence of numeric literal coefficient is
+    still the same as that of the unary operators. Hence, the expression
+    `6/2(2+1)` leads to `1` instead of the conventional result `9`, which is
```

Comment:
@KristofferC,

But, since from a mathematical point of view, it behaves very different from the well established convention, won't you think that more information to make it clear is better?

A simulator we are working was presenting a very strange result and four programmers took one day to figure out what was going on. This happened because the mathematical convention about multiplication and division precedence is very well defined. There is no problem about Julia using a different one for this case, but I think it should be crystal clear to avoid future problems.